### DISCUSSION TITLE
osu! hit objects aren't dimmed when not hittable

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#20465] [Ideas]
in reference to EZ gameplay only

in stable, hitcircles appear slightly dimmed (~75% opacity) and become opaque right before being hit
this feature is absent in lazer, where hitcircles always at 100% opacity

you can compare here, focusing on only one circle immediately after it appears:
https://www.youtube.com/watch?v=IqMNGd38Sqs

not sure if this was intentional or overlooked

**Comment by peppy** [comment]
This isn't just easy mod. This is applied to hitobjects in general, and was carried over from the original nintendo DS title. They are dimmed until the hit range for them is valid (aka when they won't shake from "note lock").

I think we will probably want to apply this again. Interestingly I've never heard complaints for or against this mechanic (not in stable where it's always been present, nor lazer where it hasn't been implemented yet).
